examples/terraform/azure/variables.tf

```diff
@@ -106,11 +106,6 @@ variable "disable_kubeapi_loadbalancer" {
 
 # Provider specific settings
 
-variable "ip_sku" {
-  default     = "Basic"
-  description = "SKU to use for IP addresses"
-}
```

Comment:
We should leave this variable but default it to `Standard` so users with existing setups can eventually migrate to the new Terraform configs if they want.